src/lxc/confile.c

```diff
@@ -166,110 +168,112 @@ lxc_config_define(proc);
+	{ "lxc.init.gid",                        set_config_init_gid,                         get_config_init_gid,                         clr_config_init_gid,                        },
+	{ "lxc.init.uid",                        set_config_init_uid,                         get_config_init_uid,                         clr_config_init_uid,                        },
+	{ "lxc.init.cwd",                        set_config_init_cwd,                         get_config_init_cwd,                         clr_config_init_cwd,                        },
+	{ "lxc.keyring.session",                 set_config_keyring_session,                  get_config_keyring_session,                  clr_config_keyring_session                  },
+	{ "lxc.log.file",                        set_config_log_file,                         get_config_log_file,                         clr_config_log_file,                        },
+	{ "lxc.log.level",                       set_config_log_level,                        get_config_log_level,                        clr_config_log_level,                       },
+	{ "lxc.log.syslog",                      set_config_log_syslog,                       get_config_log_syslog,                       clr_config_log_syslog,                      },
+	{ "lxc.monitor.unshare",                 set_config_monitor,                          get_config_monitor,                          clr_config_monitor,                         },
+	{ "lxc.monitor.signal.pdeath",           set_config_monitor_signal_pdeath,            get_config_monitor_signal_pdeath,            clr_config_monitor_signal_pdeath,           },
+	{ "lxc.mount.auto",                      set_config_mount_auto,                       get_config_mount_auto,                       clr_config_mount_auto,                      },
+	{ "lxc.mount.entry",                     set_config_mount,                            get_config_mount,                            clr_config_mount,                           },
+	{ "lxc.mount.fstab",                     set_config_mount_fstab,                      get_config_mount_fstab,                      clr_config_mount_fstab,                     },
+	{ "lxc.namespace.clone",                 set_config_namespace_clone,                  get_config_namespace_clone,                  clr_config_namespace_clone,                 },
+	{ "lxc.namespace.keep",                  set_config_namespace_keep,                   get_config_namespace_keep,                   clr_config_namespace_keep,                  },
+	{ "lxc.namespace.time.offset.boot",	 set_config_namespace_time_offset_boot,       get_config_namespace_time_offset_boot,       clr_config_namespace_time_offset_boot,      },
+	{ "lxc.namespace.time.offset.monotonic", set_config_namespace_time_offset_monotonic,  get_config_namespace_time_offset_monotonic,  clr_config_namespace_time_offset_monotonic, },
```

Comment:
Shouldn't that just be `lxc.time.offset.boot` and `lxc.time.offset.monotonic`?

Reply:
Yeah, I'm kinda torn. I went back and forth. But yeah, we could.

Reply:
`lxc.namespace` seems about namespacing sharing/attaching. Normal namespace configuration has traditionally been done at the toplevel (`lxc.mount`, `lxc.net`, `lxc.idmap`, ...)